stdlib/Statistics/src/Statistics.jl

```diff
@@ -42,9 +42,9 @@ julia> mean(skipmissing([1, missing, 3]))
 mean(itr) = mean(identity, itr)
 
 """
-    mean(f::Function, itr)
+    mean(f::Function, itr, [dims = 1])
```

Comment:
Please put this in another docstring, as `dims` is only supported for `AbstractArray`. I guess it would be OK to add a second signature to the `mean(f, A::AbstractArray)` docstring. (This docstring could also be merged with the `mean(itr)` one, but that's another story.)

Reply:
Sure